modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -225,6 +225,50 @@ TEST_P(Test_ONNX_layers, Gather)
     testONNXModels("gather", npy, 0, 0, false, false);
 }
 
+TEST_P(Test_ONNX_layers, RAFT)
+{
+    std::string weight_path = _tf("models/optical_flow_estimation_raft_2023aug.onnx", false);
+    std::string img0_path = findDataFile(std::string("gpu/opticalflow/frame0.png"));
+    std::string img1_path = findDataFile(std::string("gpu/opticalflow/frame1.png"));
+
+    Size target_size{480, 360};
+    auto img0 = imread(img0_path);
+    auto img1 = imread(img1_path);
+    cvtColor(img0, img0, COLOR_BGR2RGB);
+    cvtColor(img1, img1, COLOR_BGR2RGB);
+    auto blob0 = blobFromImage(img0);
+    auto blob1 = blobFromImage(img1);
+
+    auto net = readNet(weight_path);
+    net.setInput(blob0, "0");
+    net.setInput(blob1, "1");
+    auto out0 = net.forward("12007");
+    auto out1 = net.forward("12006");
```

Comment:
Forward model only once and request both outputs

Reply:
Done.